src/components/base/RadioGroup/RadioGroup.tsx

```
@@ -23,8 +23,8 @@ export default function RadioGroup({ options, value, onChange }: RadioGroupProps
   return (
     <RadioGroupFrame>
       {options.map((option, i) => (
-        <YStack gap={16}>
-          <XStack key={option.value} gap="$2" group onPress={handlePress(option.value)} alignItems="center" cursor="pointer">
+        <YStack gap="$medium">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)